routes/stations.js

```diff
@@ -18,6 +18,18 @@ router.get('/:_id', function(req, res, next) {
   });
 });
 
+/* GET stations given network _id */
+router.get('/network/:network_id', function(req, res, next) {
+  Station.find({"network_id": mongoose.Types.ObjectId(req.params.network_id)}, function(err, station) {
+    if(err) {
+      console.log(err);
```

Comment:
For APIs. what you would normally want is a generic response that will return something like

```
{
  status: "ok/error",
  result/restuls: 
}
```

Definitely not a requirement here but something to have in mind.